markdown/apigee-apim-integration/apigee-apim-integration.md

```diff
@@ -0,0 +1,558 @@
+author: Swen-Helge Huber
+summary:
+id: apigee-apim-integration
+tags: iguide
+categories: solace
+environments: Web
+status: Published
+feedback link: https://github.com/SolaceDev/solace-dev-codelabs/blob/master/markdown/apigee-apim-integration
+
+# Unified APIM: Integrating Solace Event Portal with apigee Platform and Developer Portal 
```

Comment:
uppercase apigee here

Reply:
There;s a couple more references to apigee that will need to be uppercased throughout the document as well

Reply:
There were a lot more. I think I have uppercased all now.